integration-tests/hibernate-search-orm-opensearch/src/main/resources/application.properties

```diff
@@ -11,4 +11,4 @@ quarkus.hibernate-search-orm.elasticsearch.hosts=${opensearch.hosts:localhost:92
 quarkus.hibernate-search-orm.elasticsearch.protocol=${opensearch.protocol:http}
 quarkus.hibernate-search-orm.elasticsearch.analysis.configurer=bean:backend-analysis
 quarkus.hibernate-search-orm.schema-management.strategy=drop-and-create-and-drop
-quarkus.hibernate-search-orm.automatic-indexing.synchronization.strategy=sync
+quarkus.hibernate-search-orm.indexing.plan.synchronization.strategy=sync
```

Comment:
Probably something we should include in the update rules, once integrated? Or maybe we keep it for the 3.3 upgrade as we are sure you will require it?

Reply:
Yes let's do it in 3.3 and then we can backport. I guess I should submit a PR to https://github.com/quarkusio/quarkus-updates ?

Reply:
I will do it.